src/sql/connection.py

```diff
@@ -3,8 +3,29 @@
 
 import sqlalchemy
 from sqlalchemy.engine import Engine
+from sqlalchemy.exc import NoSuchModuleError
 from IPython.core.error import UsageError
 
+PLOOMBER_SUPPORT_LINK_STR = (
+    "For technical support: https://ploomber.io/community"
+    "\nDocumentation: https://jupysql.ploomber.io/en/latest/connecting.html"
+)
+
+MISSING_PACKAGE_LIST_EXCEPT_MATCHERS = {
```

Comment:
yeah, this makes sense. 

why does duckdb contain `sqlalchemy.dialects` but the other doesn't? is this just how sqlalchemy displays the error?